core/api/export/cp_data_extraction_all.py

```diff
@@ -97,6 +97,13 @@ def _write_record_row(self, row_idx, country_name, chemical_name, record):
                     value = float(value)
                 except (TypeError, ValueError):
                     pass
+            elif "fob" in header_id or "retail" in header_id:
```

Comment:
I think it’s better to have this logic in the view. This way, it will be easier for us to check where and how the value is calculated in the future.
Also, we need this logic in export report too 😇